rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -22,14 +24,18 @@ def connect(addr: str | None = None, recording: RecordingStream | None = None) -
     ----------
     addr
         The ip:port to connect to
+    disconnected_timeout_sec: float
+        The minimum time the `TcpSink` will wait during a flush before potentially
```

Comment:
Coming in from this side a user has no idea what `TcpSink` means here (the Rust API at least has links that make it easy to look up what is meant)

```suggestion
        The minimum time the SDK will wait for a successful connection before potentially
```